aquasec/resource_registry.go

```diff
@@ -83,10 +83,43 @@ func resourceRegistry() *schema.Resource {
 				Optional:    true,
 				Computed:    true,
 			},
+			"scanner_name": {
+				Type:        schema.TypeList,
+				Description: "List of scanner names",
+				Optional:    true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
+			"scanner_name_added": {
+				Type:        schema.TypeList,
```

Comment:
Are the added, removed are only internal or user can set them manual, if its internal you can removed them from the scheme

Reply:
only internal, users input is not required. I tried adding them as 'computed' but failing test cases, as the terraform plan was not empty and expecting some values. In real the creation api does not have this variables.

Reply:
you don't need to add them to the scheme at all

Reply:
thanks, removed from the schema